inhouse_bot/common_utils/get_player.py

```diff
@@ -0,0 +1,22 @@
+from inhouse_bot.database_orm import Player
+from inhouse_bot.database_orm.session.session import Session, session_scope
+
+
+def get_player(
+    player_id: int, server_id: int, session: Session | None
+) -> Player | None:
+    """
+    Returns the player with the given id and server id
+    """
+
+    if session is None:

```

Comment:
This piece seems questionable

Reply:
I've done this before in queue_handler.py:add_player and score_game.py

It's useful if you want the query part of another session. Otherwise, you can leave it as none and it will make a new one-off session

Reply:
I'm removing it from here because this fn is only using it without a session

Reply:
Yea you can leave as is, but it doesn't feel right to have this recursive-esque solutions to this, it should be like one place that monitors if a session is None